src/main/java/it/pagopa/ecommerce/commons/documents/v1/TransactionAuthorizationCompletedData.java

```diff
@@ -25,6 +27,12 @@ public class TransactionAuthorizationCompletedData {
     @Nullable
     private String rrn;
 
+    /**
+     * PGS authorization completion timestamp
+     */
+    @Nullable
+    private OffsetDateTime timestampOperation;
```

Comment:
Change this field to a String just as done for [event timestamp](https://github.com/pagopa/pagopa-ecommerce-commons/blob/e330d285605efdd732bab90e5296a34c3d800d34/src/main/java/it/pagopa/ecommerce/commons/documents/v1/TransactionEvent.java#L35)(avoiding mongo serialization/deserialization issues) see